great_expectations/data_context/store/tuple_store_backend.py

```diff
@@ -76,6 +76,14 @@ def __init__(  # noqa: PLR0913
             self.verify_that_key_to_filepath_operation_is_reversible()
             self._fixed_length_key = True
 
+    @staticmethod
```

Comment:
I'm ok if we remove the `@staticmethod` decorator.

Reply:
If it isn't relying on instance or class attributes it's better as a static method.